docs/guide/relationships.md

```diff
@@ -39,6 +39,18 @@ module.exports = {
 };
 ```
 
+```django
+{# modules/article-page/views/show.html #}
+{% extends "layout.html" %}
+
+{% block main %}
+  <h1>{{ data.piece.title }}</h1>
+  <section>
+    {% area data.piece, 'body' %}
+  </section>
+{% endblock %}
+```
+
```

Comment:
you may not have to create this file, since it's the default show page, and it should be used automatically by the default core `piece-page-type` module

Reply:
It will be difficult for a newcomer to know how to render a rich-text widget.

It is mentioned later in this page to add content to the file without showing what should be there in the first place